runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -383,7 +384,9 @@ public void onChatMessage(ChatMessage event)
 			return;
 		}
 
-		if (config.showStamina() && event.getMessage().equals(STAMINA_DRINK_MESSAGE))
+		if (config.showStamina()
+			&& event.getMessage().equals(STAMINA_DRINK_MESSAGE)
+			|| event.getMessage().equals(STAMINA_SHARED_DRINK_MESSAGE))
```

Comment:
I would group these statements so they are explicit on what is happening.

Reply:
Hmm I'm confused about what you mean by "group". Can you clarify?

Reply:
parentheses around the second two